docs/en/observability/quickstarts/monitor-k8s-otel.asciidoc

```diff
@@ -0,0 +1,65 @@
+[[monitor-k8s-otel-edot]]
+= Quickstart: Unified Kubernetes Observability with Elastic Distributions of OpenTelemetry (EDOT)
+
+preview::[]
+
+In this quickstart guide, you will learn how to send Kubernetes logs, metrics, and application traces to Elasticsearch using the https://github.com/open-telemetry/opentelemetry-operator/[OpenTelemetry Operator] to orchestrate the https://github.com/elastic/opentelemetry/tree/main[Elastic Distributions of OpenTelemetry] (EDOT) Collectors and SDK instances you will need.
+
+All the components will be deployed through the https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-kube-stack[opentelemetry-kube-stack] helm chart. They include:
+
+* https://github.com/open-telemetry/opentelemetry-operator/[OpenTelemetry Operator]
+* `DaemonSet` EDOT Collector configured for node level metrics.
+* `Deployment` EDOT Collector configured for cluster level metrics.
+* Instrumentation object for applications https://opentelemetry.io/docs/kubernetes/operator/automatic/[auto-instrumentation].
```

Comment:
There is inconsistent punctuation used in this list. Maybe add a period to the first item